packages/monaco/src/browser/monaco-icon-registry-types.ts

```diff
@@ -0,0 +1,127 @@
+// *****************************************************************************
+// Copyright (C) 2023 Ericsson and others.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { Event } from '@theia/core';
+import { IJSONSchema } from '@theia/core/lib/common/json-schema';
+import { ThemeIcon } from '@theia/monaco-editor-core/esm/vs/platform/theme/common/themeService';
+import { URI } from '@theia/core/shared/vscode-uri';
+
+//  ------ API types
+
+export type IconIdentifier = string;
+
+// icon registry
+export const Extensions = {
+    IconContribution: 'base.contributions.icons'
+};
+
+export type IconDefaults = ThemeIcon | IconDefinition;
```

Comment:
This seems to be the type that happens to be used where a default value for an icon is needed. For this, the name is misleading. The type is used exactly once and it does not describe anything, so I would just remove it.